docs/source/user/getting-started.rst

```diff
@@ -4,32 +4,145 @@ Getting Started
+* macOS, x86 64-bit, version 10.13 or later
+* Windows, x86 64-bit, version 7 or later
+
+Other platforms may work, but will require `building from source <#building-opendp-from-source>`_.
+
+Installing OpenDP for Python from PyPI
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+OpenDP is distributed as a `Python package on PyPI <https://pypi.org/project/opendp/>`_. You can install it using ``pip`` or another package management tool:
+
+.. prompt:: bash
+
+    pip install opendp
+
+.. note::
+
+    The OpenDP Python package contains binaries for all supported platforms, so you don't need to worry about specifying a specific platform.
+
+OpenDP has no dependencies, so at this point, you should be good to go! You can confirm your installation in Python by importing the top-level module ``opendp``:
+
+.. doctest::
+
+    >>> import opendp
+
+Installing OpenDP for Rust from crates.io
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+OpenDP is also available as a `Rust crate <https://crates.io/crates/opendp>`_. This is uncommon, as most people will use OpenDP through one of the other language bindings. But if you need to use the Rust interface of OpenDP directly, you can just reference the ``opendp`` crate in your ``Cargo.toml`` file:
+
+.. code-block:: toml
+
+    [dependencies]
+    opendp = "0.1.0"
+
+.. note::
+
+    The actual version may differ depending on the `releases available <https://github.com/opendp/opendp/releases>`_.
+
+With that configured, the Rust dependency system will automatically download the crate as needed, and you can just ``use`` the ``opendp`` module:
+
+.. code-block:: rust
+
+    use opendp::core::*;
+    // OpenDP code goes here!
 
-Installing OpenDP from PyPI
+Building OpenDP from Source
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-A package for OpenDP is available from `PyPI <https://pypi.org/project/opendp/>`_. You can install it using ``pip`` or other package management tool:
+Under special circumstances, it may be necessary to install OpenDP directly from the source files. This is only required if you want to build OpenDP from scratch, or if you're interested in :doc:`contributing to OpenDP <../developer/contributing>`.
 
-.. code-block:: bash
+For this to work, you'll need some prerequisites:
 
-    % pip install opendp
+* The `Rust toolchain <https://www.rust-lang.org/tools/install>`_, edition 2018.
+* Python, version 3.6 or higher.
+* Python package `pytest <https://docs.pytest.org/en/stable/>`_ (optional but recommended).
 
-This will make the OpenDP modules available to your local environment.
+Assuming your base environment is ready, start by cloning the `opendp git repo <https://github.com/opendp/opendp>`_:
+
+.. prompt:: bash
+
+    git clone git@github.com:opendp/opendp.git
+    cd opendp
+
+Next, you'll need to build the Rust binaries. This is done by running ``cargo build`` in the ``rust`` subdirectory of the repo:
+
+.. prompt:: bash
+
+    cd rust
+    cargo build
```

Comment:
It would be good to add a line below for Windows users (that there is a flag you can add to hide the pain, and where you can find the complicated process `/windows` to heal the pain)

Reply:
Great suggestion, added.